desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -446,6 +446,12 @@ class OnePageViewModel {
       self.loadApp('importer');
       self.getActiveAppViewModel(vm => {
         vm.createWizard.source.path(path);
+        vm.currentStep(2);
+        vm.createWizard.source.interpreter('hive');
+        vm.createWizard.destination.name('default.xyz1');
```

Comment:
When we click on the + in the left panel we get those values, would reusing the same logic work here too?